arlas-server/src/main/java/io/arlas/server/impl/ElasticDatabaseToolsFactory.java

```diff
@@ -41,16 +41,20 @@
     private final ElasticClient elasticClient;
     private final ExploreService exploreService;
     private final CollectionReferenceDao collectionReferenceDao;
-    private final OGCCollectionReferenceDao ogcDao;
-    private final WFSToolService wfsService;
+    private OGCCollectionReferenceDao ogcDao;
+    private WFSToolService wfsService;
 
     public ElasticDatabaseToolsFactory(ArlasServerConfiguration configuration) {
         super(configuration);
         this.elasticClient = new ElasticClient(configuration.elasticConfiguration);
-        this.exploreService = new ElasticExploreService(elasticClient, configuration);
         this.collectionReferenceDao = new ElasticCollectionReferenceDao(elasticClient, configuration.arlasindex, configuration.arlascachesize, configuration.arlascachetimeout);
-        this.ogcDao = new ElasticOGCCollectionReferenceDao(elasticClient, collectionReferenceDao, configuration.arlasindex, Service.CSW);
-        this.wfsService = new ElasticWFSToolService((ElasticExploreService) exploreService);
+        this.exploreService = new ElasticExploreService(elasticClient, collectionReferenceDao, configuration.arlasBaseUri, configuration.arlasrestcachetimeout);
```

Comment:
also this one isn't camel cased

Reply:
This one comes from the configuration class which I haven't touched. I could fix it though, as there are many variables in it which are not camel case.

Reply:
Done